doc/articles/get-started-dotnet-new.md

```diff
@@ -134,9 +134,9 @@ dotnet new unoapp-net6 -o MyApp -mobile=false
 
 ## Uno Platform Blank Application for WinAppSDK - WinUI 3 (.NET 6 - Preview)
 
-This template can be used to create a blank multi-platform application for iOS, Android, WebAssembly, macOS, mac Catalyst, Linux, and Win32 Desktop which uses the new WinUI 3 apis.
+This template can be used to create a blank multi-platform application for iOS, Android, WebAssembly, macOS, Mac Catalyst, Linux, and Win32 Desktop which uses the new WinUI 3 apis.
```

Comment:
```suggestion
This template can be used to create a blank multi-platform application for iOS, Android, WebAssembly, macOS, Mac Catalyst, Linux, and Win32 Desktop which uses the new WinUI 3 APIs.
```

Reply:
Thanks! I'll be pushing this as is and will make the change in the next PR!